mdsa/pcoa.py

```diff
@@ -95,7 +95,8 @@ def pcoa(distance_matrix, algorithm, num_dimensions_out=10):
     eigenvectors, eigenvalues = algorithm.run(F_matrix, num_dimensions_out)
 
     # Ensure eigenvectors are normalized
-    eigenvectors = [vec / np.linalg.norm(vec) for vec in eigenvectors]
+    eigenvectors = np.array([vec / np.linalg.norm(vec)
```

Comment:
Wondering if this could simply be: 
`eigenvectors = eigenvectors / np.linalg.norm(vec)`
or
`eigenvectors = np.array(eigenvectors) / np.linalg.norm(vec)`